main.go

```diff
@@ -56,6 +62,8 @@ import (
 const (
 	flagHealthProbeBindAddr       = "health-probe-bind-addr"
 	defaultHealthProbeBindAddress = ":61779"
+	defaultAPIServerQPS           = 1e6
```

Comment:
Is there a reason why we are adding such high default QPS? I believe this could negatively impact other components including API Server if this controller is misbehaving for some reason.

Reply:
These values are from from the loadbalancer repo